README.md

```diff
@@ -31,13 +31,30 @@ Run all service on local machine:
 
     npm run api-layer
 
-## Setting up connection to z/OSMF
+## Authentication service
 
-z/OSMF is required for the API Catalog login. You need to configure a valid z/OSMF instance to `config/local/api-defs/zosmf-sample.yml` under the `instanceBaseUrls`.
+API Mediation Layer uses dummy authentication provider as the default security provider for the development purposes. In order to login,
```

Comment:
The API Mediation Layer uses a dummy authentication provides as a default security provider for the development purposes. To log in, use "user" as username and password.